tests/test_radarr.py

```diff
@@ -0,0 +1,785 @@
+import contextlib
```

Comment:
We might want to consider linking this repo to [`codecov`](https://about.codecov.io/) to get pretty/user-readable coverage printouts on each PR.

Reply:
Yes, I wanted to get this release done then will add to code cov.

Reply:
In order to avoid static fixtures and have more robust testing, it may make sense to setup a test instance of Sonarr/Radarr/etc during testing.

This could be accomplished one of three ways
1) Directly storing a linux `-arr` pre-built executables within the `tests` directory.
2) Store pre-built executables within separate PyArr branches, then [download them from that GitHub branch](https://stackoverflow.com/questions/43520843/how-to-clone-from-specific-branch-from-git-using-gitpython) during testing, if they're not already downloaded. Would require you to commit the `-arr` DB in order to keep API keys consistent across test runs.
3) Store only the `-arr` DBs within this repo then [install the -arrs](https://sonarr.tv/#downloads-v3-linux) the normal way, then transfer the database over once the install completes.

Then, the `-arrs` would need to be spawned during tests via [`subprocess.run`](https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module) through PyTest or a Nox..